### PR TITLE
Miscellaneous

### PR DESCRIPTION
**Pull request by aalhossary**

### CODE REVIEW

File: biojava-structure/src/main/java/org/biojava/nbio/structure/HetatomImpl.java [hunk 1]
```diff
@@ -168,7 +168,7 @@ public void addAtom(Atom atom){
 				String altLocStr = "";
 				char altLoc = atom.getAltLoc();
 				if (altLoc != ' ') altLocStr = "(alt loc '" + altLoc + "')";
-				logger.warn("An atom with name " + atom.getName() + " " + altLocStr + " is already present in group: " + this.toString() + ". The atom with serial " + atom.getPDBserial() + " will be ignored in look-ups.");
+				logger.warn("An atom with name " + atom.getName() + " " + altLocStr + " is already present in group: " + this.toString() + ". The atom with serial " + existingAtom.getPDBserial() + " will be ignored in look-ups.");
```
Comment (josemduarte) [inline biojava-structure/src/main/java/org/biojava/nbio/structure/HetatomImpl.java:171]: Good catch, thanks!

File: biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/FastaGeneWriterTest.java [hunk 2]
```diff
@@ -58,7 +58,7 @@ void basicGeneWriterTest() throws Exception {
         fastaWriter.process();
 
         String output = new String(os.toByteArray(), "UTF-8");
-        String [] lines =  output.split("\n");
+        String [] lines =  output.split("\r?\n");
```
Comment (josemduarte) [inline biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/FastaGeneWriterTest.java:61]: How about using `\R`?, see https://www.baeldung.com/java-string-split-by-newline

**Review by josemduarte — CHANGES_REQUESTED** [review]
Thanks!

**Review by josemduarte — APPROVED** [review]
Thanks! LGTM

**Comment by aalhossary** [comment]
Thank you for the tip. I didn't know about \R before.

Do you have any idea why it failed twice?

**Comment by josemduarte** [comment]
It looks like there are problems with the SCOP servers. I'll try it again tomorrow.

**Comment by aalhossary** [comment]
I triggered a new build which was successful this time. Will merge now.

**Pull request merged**